lib/cisco_node_utils/cmd_ref/interface_hsrp_group.yaml

```diff
@@ -1,6 +1,6 @@
 # interface_hsrp_group
```

Comment:
When I configure an hsrp group on my n9k, the default value is `cisco`.  Not sure I understand why it's 'cleartext' here.

https://github.com/cisco/cisco-network-node-utils/blob/3a8e8c5d4339e5faf5ab744cc413d23f3b59f3bc/lib/cisco_node_utils/cmd_ref/interface_hsrp_group.yaml#L16